server/etcdserver/apply.go

```diff
@@ -148,15 +148,15 @@ func (a *applierV3backend) Apply(r *pb.InternalRaftRequest, shouldApplyV3 member
 	case r.ClusterVersionSet != nil: // Implemented in 3.5.x
 		op = "ClusterVersionSet"
 		a.s.applyV3Internal.ClusterVersionSet(r.ClusterVersionSet, shouldApplyV3)
-		return nil
+		return &applyResult{}
```

Comment:
Just a nit, There is already a variable at line 137, so we can return `ar` directly.

Reply:
Done